src/openapi-generator.ts

```diff
@@ -632,6 +632,13 @@ export class OpenAPIGenerator {
       const regexCheck = this.getZodStringCheck(zodSchema, 'regex');
       return {
         type: 'string',
+        // FIXME: https://github.com/colinhacks/zod/commit/d78047e9f44596a96d637abb0ce209cd2732d88c
```

Comment:
Hmm that is interesting. Great catch. I am not sure if I want to raise the minimal zod version for this check only. So I would rather keep this in its current state for now (using `isFinite`). Once we get to update for a different reason we can revise this. cc: @georgyangelov any thoughts?

Reply:
You can probably potentially just bump up your dev deps and pass in `zodSchema.minLength` since it won't be defined in previous versions it'll end up as just undefined?

Reply:
Uhmmm @samchungy  from what I saw both `minLength` and `maxLength` exist in the current dependency version (3.14.3). The only difference being the default that is being used - `null` for older versions and `Infinity` for the latest zod version. And seeing that we cannot pass an infinity to openapi3ts -> I think that `isFinite` is the correct check overall

Reply:
@wiz-kakiuchi do you mind resolving the conflicts and possibly fixing the nullish coalescence above.